dali/base/dadfs.cpp

```diff
@@ -3143,6 +3143,34 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
         setAccessedTime(dt);
     }
 
+    virtual void addAttrValue(const char *attr, unsigned __int64 value) override
+    {
+        if (0==value)
+            return;
+        if (logicalName.isForeign())
+        {
+            // Note: foreign files will not be updated.  Place holder here if the need every arises.
```

Comment:
I think this is fine for now, perhaps reword, it's not that it's not needed, it's that's not possible to implement at moment.
We would update foreign if we could, but it will have to wait I think until DFS is refactored so it is via esp, at which point it can relatively easily go to any esp including foreign esp's